batch-nft-reveal/test/NFTCollection.test.ts

```diff
@@ -172,7 +172,8 @@ describe("NFTCollection", async function () {
         "./test/data/unrevealed_metadata.txt",
         "utf8"
       );
-      const uri = await nftCollection.tokenURI(0);
+      await mint(nftCollection, NFT_REVEAL_BATCH_SIZE);
```

Comment:
The first part of a unit test is to do the setup. Check the other tests. Also add new lines like the other tests for consistency.